WordPress/Classes/Utility/WPTableViewHandler.m

```diff
@@ -151,15 +151,18 @@ - (void)refreshTableViewPreservingOffset
         }
         newIndexPath = [self.resultsController indexPathForObject:obj];
 
-        CGRect originalCellFrame = [[visibleCellFrames objectAtIndex:i] CGRectValue];
-        if (newIndexPath) {
-            // Since we still have one of the originally visible objects,
-            // preserve the original cell frame's origin, relative to the original content offset.
-            originalCellOriginOffsetDelta += originalOffset.y - originalCellFrame.origin.y;
-            break;
-        } else {
-            originalCellOriginOffsetDelta -= originalCellFrame.size.height;
+        if (visibleCellFrames.count >= i) {
```

Comment:
If `i` is higher than the count, shouldn't we `break` out of the loop instead of continue iterating?

Reply:
I leave without it just to keep the code without too many branches (`if`, `else`, `if`, `else`). It will probably iterate there a few more times but I'm not sure if it makes any performance difference.

What do you think? With or without it?